app/routes/press-releases/overview/published.js

```diff
@@ -1,4 +1,22 @@
 import Route from '@ember/routing/route';
+// eslint-disable-next-line ember/no-mixins
+import DataTableRouteMixin from 'ember-data-table/mixins/route';
 
-export default class PressReleasesOverviewPublishedRoute extends Route {
-}
+
+export default class PressReleasesOverviewPublishedRoute extends Route.extend(DataTableRouteMixin) {
+  modelName = 'press-release';
+
+  mergeQueryOptions(params) {
+    const queryParams = {
+      sort: params.sort,
+      include: 'publication-event',
+      filter: {
+        ':has:publication-event': 'yes',
+      },
+    };
+
+    queryParams['filter[publication-event][:has:started]'] = 'yes'
```

Comment:
De :has: filter kan enkel gebruikt worden voor relaties, niet voor attributes. In de documentatie van de API staan alle speciale filters toegelicht. Om te kijken of een datum in het verleden ligt, moet je gebruik maken van :lt: (less than) en vergelijken met de huidige datum/tijd.

Is er een specifieke reden om deze filter op een aparte lijn te definiëren en niet mee op te nemen in het queryParams object hierboven?